uint8 FULL_VIEW=0
uint8 PARTIAL_VIEW=1

string name
uint8 view
bool is_open
geometry_msgs/Point position